platform/spire/src/command.py

```diff
@@ -315,7 +315,12 @@ def __call__(self, depth=0, dry_run=False):
 
 
 def main_invoke(command):
-    parser = argparse.ArgumentParser(description="Administrative toolkit for deploying and maintaining Hyades clusters")
+    # we have to hardcode the name "spire" here, because using bazel's py_binary tool for packaging our program into a
+    # zip file means we'll get the name of our __main__.py, rather than the name of the original command invoked.
```

Comment:
should we file a bug against rules_python for this?

Reply:
I'm not sure there's much they can do about it? They have to pass a particular name to python in order to get the right script to run. I don't think they can pass the original name without it just not working -- unless they add another layer of stub script that runs within the target python interpreter, but that might mess up other semantics.

We could file an issue on upstream or on our own repo so that we remember this later, but I don't think this is a real problem for us. If we ever rename spire, we can just change this piece of code to reflect the new name.